app/factory.py

```diff
@@ -43,23 +43,27 @@ def get_user(self, identifier):
             if rv is not None:
                 return rv
 
+user_datastore = DeploySQLAlchemyUserDatastore(db, User, Role)
```

Comment:
Hmm, I wonder if it'd be useful to move `user_datastore` and `DeploySQLAlchemyUserDatastore` to `models.py`...